### PR TITLE
Configuring Kubernetes Use of etcd--replace / to or

### PR DESCRIPTION
**Pull request by jianglingxia**
read/write access to etcd -- change to read or write , or read and write?

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2651)

### CODE REVIEW

File: docs/admin/etcd.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ objects.
 
 ## Configuration: high-level goals
 
-Access Control: give *only* kube-apiserver read/write access to etcd. You do not
```
Comment (chenopis) [inline docs/admin/etcd.md]: "read/write" is a standard convention for saying "read and write" so I would leave that unchanged.

Comment (jianglingxia) [inline docs/admin/etcd.md]: ok ,I will close it ,thanks

**Comment by chenopis** [comment]
@jianglingxia Sorry, I should have also mentioned that the other change is still needed. Feel free to still add the code format for "$ kubectl describe TYPE NAME_PREFIX" in docs/user-guide/kubectl/kubectl_describe.md

**Comment by jianglingxia** [comment]
thanks for you that reopen ,but I have pull another request  to :https://github.com/kubernetes/kubernetes.github.io/pull/2678 that the same problem ,you can review it ,thanks !

**Comment by chenopis** [comment]
@jianglingxia Ok, great. Thanks!

**Pull request closed without merging**